src/main/java/TaskList.java

```diff
@@ -0,0 +1,57 @@
+public class TaskList {
+    Task[] list = new Task[100];
+    private int index = 0;
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+    public void unmarkStatus(String input) {
+        String[] instructions = input.split(" ");
+        int number = Integer.parseInt(instructions[1]);
+        list[number-1].isDone = false;
+        printLine();
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println("  [ ] " + list[number-1].description);
+        printLine();
+    }
+
+    public void markStatus(String input) {
+        String[] instructions = input.split(" ");
+        int number = Integer.parseInt(instructions[1]);
+        list[number-1].isDone = true;
+        printLine();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("  [X] " + list[number-1].description);
+        printLine();
+    }
```

Comment:
Could be better if markStatus and unmarkStatus came from a more generic function. Both functions are very similar, hence there should be less repeat code